packages/garbo/src/resources/candyMap.ts

```diff
@@ -0,0 +1,117 @@
+import { AcquireItem, Outfit } from "grimoire-kolmafia";
+import {
+  $familiar,
+  $item,
+  get,
+  getSaleValue,
+  have,
+  sum,
+} from "libram";
+import {
+  canEquip,
+  getOutfits,
+  mallPrice,
+  outfit,
+  outfitPieces,
+  outfitTreats,
+  print,
+  runChoice,
+  toItem,
+  use,
+  visitUrl,
+} from "kolmafia";
+import { GarboTask } from "../tasks/engine";
+import { garboValue } from "../garboValue";
+
+const HOUSE_NUMBERS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+
+let blockHtml = "";
+
+function getBlockHtml(): string {
+  blockHtml ||= visitUrl("place.php?whichplace=town&action=town_trickortreat");
+  return blockHtml;
+}
+
+function treatValue(outfit: string): number {
+  return sum(
+    Object.entries(outfitTreats(outfit)),
+    ([candyName, probability]) => probability * garboValue(toItem(candyName))
+  );
+}
+
+export function getTreatOutfit(): string {
+  const availableOutfits = getOutfits().filter((name) =>
+    outfitPieces(name).every((piece) => canEquip(piece))
+  );
+  if (!availableOutfits.length) {
+    print("You don't seem to actually have any outfits available, my friend!");
+  }
+  return maxBy(availableOutfits, treatValue);
+}
+
+export function treatOutfit(): Outfit {
+  const outfit = new Outfit();
+  const pieces = outfitPieces(getTreatOutfit());
+  for (const piece of pieces) {
+    if (!outfit.equip(piece))
+      print(`Could not equip all pieces of treat outfit: aborted on ${piece}`);
+  }
+
+  outfit.equip($item`lucky Crimbo tiki necklace`);
```

Comment:
We as a society have decided that this does nothing. I'll remove it from freecandy if relevant

Reply:
Removed, then.